packages/keepalived/packaging

```diff
@@ -0,0 +1,12 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+tar xzvf keepalived/keepalived-1.2.24.tar.gz
```

Comment:
discussion about blobs should appear in blobs.yml not on this file @gberche-orange